tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+import wazuh_testing.fim as fim
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
+sub_key_1 = "SOFTWARE\\Classes\\testkey"
+sub_key_2 = "SOFTWARE\\Classes\\Testkey"
+
+test_regs = [os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)]
+wazuh_log_monitor = FileMonitor(fim.LOG_FILE_PATH)
+test_data_path = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), 'data')
+reg1, reg2 = test_regs
+
+monitoring_modes = ['scheduled']
+
+# Configurations
+
+conf_params = {'WINDOWS_DUPLICATED_REGISTRY_1': reg1,
+               'WINDOWS_DUPLICATED_REGISTRY_2': reg2}
+configurations_path = os.path.join(
+    test_data_path, 'wazuh_conf_duplicated_registry.yaml')
+p, m = fim.generate_params(extra_params=conf_params, modes=monitoring_modes)
+configurations = load_wazuh_configurations(
+    configurations_path, __name__, params=p, metadata=m)
+
+registry_list = [(key, sub_key_1, fim.KEY_WOW64_32KEY),
+                 (key, sub_key_2, fim.KEY_WOW64_32KEY)]
+
+# fixtures
+
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+# test
+
+
+@pytest.mark.parametrize('key, subkey1, subkey2, arch', [
+    (key, sub_key_1, sub_key_2, fim.KEY_WOW64_32KEY)
+])
+def test_registry_duplicated_entry(key, subkey1, subkey2, arch,
+                                   get_configuration, configure_environment,
+                                   restart_syscheckd, wait_for_fim_start):
```

Comment:
```suggestion
@pytest.mark.parametrize('key, subkey1, subkey2, arch', [(key, sub_key_1, sub_key_2, fim.KEY_WOW64_32KEY)])
def test_registry_duplicated_entry(key, subkey1, subkey2, arch, get_configuration, configure_environment,
                                   restart_syscheckd, wait_for_fim_start):
```
You have up to 120 characters. Feel free to use them.